lib/cisco_node_utils/command_reference.rb

```diff
@@ -142,13 +142,12 @@ def valid?
 
   # Builds reference hash for the platform specified in the product id.
   class CommandReference
-    attr_reader :debug
```

Comment:
They're actually redundant ('rake test' was warning about this) - we explicitly define a `debug` function below which overwrites this attr_reader.